docs/index.rst

```diff
@@ -106,7 +106,7 @@ Installation
 
    .. tab-item:: Windows
 
-      `Download the Windows Installer <https://github.com/neuronsimulator/nrn/releases/download/8.2.4/nrn-8.2.4.w64-mingw-py-39-310-311-312-setup.exe.exe>`_.
+      `Download the Windows Installer <https://github.com/neuronsimulator/nrn/releases/download/8.2.6/nrn-8.2.6.w64-mingw-py-39-310-311-312-setup.exe.exe>`_.
```

Comment:
There's an extra `.exe` here that caused headaches before. Also, I don't know what the supported Python versions are (I think it should also have `38` since it's not EOL yet).

Reply:
I have a vague memory of 3.8 being explicitly removed here, so I'd rather not add it back. I can definitely remove the extra `.exe` though 🤦

Reply:
There's this commit: https://github.com/neuronsimulator/nrn/commit/d6c714c47adeb1923ec0614f63136db97226d8ab
but I can't seem to find a reason why we don't support 3.8 anymore, at least for the Windows installer 🤔